### PR TITLE
Fix storybook preview

### PR DESCRIPTION
**Pull request by jasonLaster**

### CODE REVIEW

File: src/components/stories/Preview.js [hunk 1]
```diff
@@ -139,26 +139,29 @@ function PreviewFactory(options, { dir = "ltr", theme = "light" } = {}) {
...
-          loadObjectProperties: () => {},
-          onClose: action("onClose")
-        },
-        options
+    { className: "editor-wrapper" },
+    dom.div(
+      {
+        className: `preview ${themeClass}`,
+        dir,
+        style: {
```
Comment (codehag) [inline src/components/stories/Preview.js]: any chance this can be moved into a class?

Comment (jasonLaster) [inline src/components/stories/Preview.js]: hmm - we could, but it's a story so i would prefer to keep the styles here so it is clear. 

We have a stylesheet for another story, so we do it both ways

Comment (codehag) [inline src/components/stories/Preview.js]: cool, thanks for clarifying!

Comment (jasonLaster) [inline src/components/stories/Preview.js]: here's the css for another story: https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-splitter/stories/index.css#L14

**Pull request merged**